systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java

```diff
@@ -262,16 +259,14 @@ public void testRackAwareConnectCorrectDeployment(ExtensionContext extensionCont
         // We need to update CO configuration to set OPERATION_TIMEOUT to shorter value, because we expect timeout in that test
         Map<String, String> coSnapshot = DeploymentUtils.depSnapshot(ResourceManager.getCoDeploymentName());
         // We have to install CO in class stack, otherwise it will be deleted at the end of test case and all following tests will fail
-        resourceManager.createResource(sharedExtensionContext,
-            new BundleResource.BundleResourceBuilder()
-                .withName(Constants.STRIMZI_DEPLOYMENT_NAME)
-                .withNamespace(NAMESPACE)
-                .withWatchingNamespaces(NAMESPACE)
-                .withOperationTimeout(CO_OPERATION_TIMEOUT_SHORT)
-                .withReconciliationInterval(Constants.RECONCILIATION_INTERVAL)
-                .buildBundleInstance()
-                .buildBundleDeployment()
-                .build());
+        install = new SetupClusterOperator.SetupClusterOperatorBuilder()
+            .withExtensionContext(extensionContext)
+            .withNamespace(NAMESPACE)
+            .withWatchingNamespaces(NAMESPACE)
```

Comment:
You can remove this `.withWatchingNamespaces(NAMESPACE)` because inside the constructor we assign default values which in this case would be `NAMESPACE`